x-pack/test/functional/apps/lens/group5/heatmap.ts

```diff
@@ -130,10 +130,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       // assert legend has a rounded value
       expect(debugState?.legend!.items).to.eql([
-        { key: '5,722.775 - 8,529.2', name: '5,722.775 - 8,529.2', color: '#61a2ff' },
-        { key: '8,529.2 - 11,335.66', name: '8,529.2 - 11,335.66', color: '#c8deff' },
-        { key: '11,335.66 - 14,142.1', name: '11,335.66 - 14,142.1', color: '#f6f9fc' },
-        { key: '14,142.1 - 16,948.55', name: '14,142.1 - 16,948.55', color: '#ffccc6' },
+        { key: '7,747 - 8,529.2', name: '7,747 - 8,529.2', color: '#61a2ff' },
```

Comment:
Not sure why this label would change here?

Reply:
I got to the bottom of it and fixed it. Heatmap config uses a `type="number` field which doesn't work well with `clearWithKeyboard` - when you clean it, it automatically sets the value to `0` which then gets appended to any updated values when `testSubjects.setValue` is called on that field. The value changed from `5722.7747` to `7748` because when I ran this test suite locally the `.` character reset the field value due to my locale (`,` being the decimal separator and not `.`) and I blindly updated the assertion thinking the test data might've regenerated.

I removed `clearWithKeyboard` and `clearWithKeyboard` which seems to do the trick.